.changeset/cool-spies-laugh.md

```diff
@@ -0,0 +1,9 @@
+---
+'explorer': patch
+'hostd': patch
+'renterd': patch
+'walletd': patch
+'website': patch
+---
+
+Fixed an issue where scrollbars could not be grabbed by moving mouse to the edge of the screen. Closes https://github.com/SiaFoundation/hostd/issues/423 Fixed by https://github.com/kocoten1992
```

Comment:
cc @kocoten1992 we forgot to include a changeset (`npx changeset`) in the last PR, this is what bumps the UI version.